### PR TITLE
[migration] Added default operator permission #29

### LINKED ISSUE #29: Add default permissions for operators
**Reserved for GCI**

Now that https://github.com/openwisp/openwisp-users/pull/40 is ready  we should enable operators to manage the following objects:

- Link
- Node

While these objects should be read-only (will only work for django >= 2.1):

- Topology

Administrators instead will be able to manage everything.

Similar PRs done in other modules:

- https://github.com/openwisp/openwisp-users/pull/40
- https://github.com/openwisp/openwisp-controller/pull/57

### PR DESCRIPTION
**Pull request by ppabcd**
Fixes #29

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 django-netjsongraph>=0.4.2,<0.5.0
-openwisp-users>=0.1.8,<0.2
+git+git://github.com/openwisp/openwisp-users.git#openwisp-users
```
Comment (ppabcd) [inline requirements.txt]: I don't know how to get latest openwisp-users

Comment (nemesisdesign) [reply requirements.txt]: no don't do this, load it in the travis build

Comment (ppabcd) [reply requirements.txt]: ok i'll try install from travis

**Comment by ppabcd** [comment]
all done @nemesisdesign

**Pull request merged**